drivers/rpmsg/CMakeLists.txt

```diff
@@ -27,5 +27,9 @@ if(CONFIG_RPMSG)
     list(APPEND SRCS rpmsg_ping.c)
   endif()
 
+  if(CONFIG_RPMSG_VIRTIO)
```

Comment:
Should it also have:
```
if(CONFIG_RPMSG_VIRTIO_IVSHMEM)
   list(APPEND SRCS rpmsg_virtio_ivshmem.c)
endif()
```
?

Reply:
Thanks, I will update this PR later today.

Reply:
Done